src/main/java/com/example/demo/Utils/EmailSender.java

```diff
@@ -0,0 +1,44 @@
+package com.example.demo.Utils;
+
+
+import com.example.demo.Model.Email;
+import com.example.demo.ScheduledTask;
+import org.slf4j.LoggerFactory;
+import org.springframework.mail.MailException;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailSender {
+
+    private JavaMailSender mailSender;
+
+    public EmailSender(JavaMailSender mailSender) {
+        this.mailSender = mailSender;
+    }
+
+    public Boolean sendEmail(Email email){
+
+        SimpleMailMessage mailMessage = new SimpleMailMessage();
+        org.slf4j.Logger logger = LoggerFactory.getLogger(ScheduledTask.class);
```

Comment:
Creating logger is expensive operation. Please, add static initialization for this